src/js/media-controller.js

```diff
@@ -993,6 +999,15 @@ const volumeSupportPromise = hasVolumeSupportAsync().then((supported) => {
 
 const airplaySupported = !!window.WebKitPlaybackTargetAvailabilityEvent;
 
+function serializeTimeRanges(timeRanges = []) {
+  return Array.from(timeRanges)
+    .map((_, i) => [
+      Number(timeRanges.start(i).toFixed(2)),
+      Number(timeRanges.end(i).toFixed(2)),
+    ])
+    .join(' ');
```

Comment:
non-blocking: I'm cool with using `,` to separate the tuple values, but for other complex object's we've used `:`. Since this is only relevant for "advanced usage, I think it's fine if we want to start here.

Reply:
will update